ynput/OpenPype
# Fusion - Fixed "optional validation"

## Changelog Description
Added OptionalPyblishPluginMixin and is_active checks for all publish tools that should be optional

## Additional info
I did add the needed code to `increment_current_file.py` but the optional-switch doesn't show up in the publisher UI. Maybe it doesn't work for integrators?

## Testing notes:
1. Add a BG node set to something else than 32bit
2. Disable the 32bit check in publisher
3. Publish without getting stopped!
